### PR TITLE
Added separate kms keys for encryption of master and worker nodes

### PR DESCRIPTION
**Pull request by arjunrn**
Added separate keys for master and worker nodes so that worker nodes cannot decrypt files for master. Also keep the existing keys till when CLM is updated.

### CODE REVIEW

**Comment by arjunrn** [comment]
:+1:

File: cluster/cluster.yaml [hunk 1]
```diff
@@ -1302,6 +1302,74 @@ Resources:
...
+          - Sid: "Allow encryption with the key"
+            Effect: "Allow"
+            Principal:
+              AWS: "arn:aws:iam::{{.Cluster.InfrastructureAccount | getAWSAccountID}}:role/cluster-lifecycle-manager-entrypoint"
+            Action:
+            - "kms:Describe"
+            - "kms:Get*"
+            - "kms:Encrypt"
+            Resource: "*"
+          - Sid: "Enable master and worker nodes to decrypt the remote files"
```
Comment (mikkeloscar) [inline cluster/cluster.yaml]: nit: Mentions both master and worker.

**Comment by arjunrn** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**